### PR TITLE
Variable/fixed packet length mode config for RF69/SX1231

### PR DESCRIPTION
**Pull request by phretor**
Related idea: is it possible to just call `RF69::begin` within `SX1X1231::begin` and just add the `if (_chipRevision == SX1231_CHIP_REVISION_2_A)` block before returning?

### CODE REVIEW

File: src/modules/RF69.cpp [hunk 1]
```diff
@@ -250,6 +261,11 @@ int16_t RF69::startTransmit(uint8_t* data, size_t len, uint8_t addr) {
   // set packet length
   _mod->SPIwriteRegister(RF69_REG_FIFO, len);
 
+  // optionally write packet length
+  if (_packetLengthConfig == RF69_PACKET_FORMAT_VARIABLE) {
+    SPIwriteRegister(RF69_REG_FIFO, len);
```
Comment (jgromes) [inline src/modules/RF69.cpp]: Call `_mod->SPIwriteRegister()`

Comment (phretor) [reply src/modules/RF69.cpp]: OK. BTW, why some modules have `_mod` and some don't?

**Comment by phretor** [comment]
Changeset [39a4614](https://github.com/jgromes/RadioLib/pull/56/commits/39a4614a775f0f81fd00cafb4af4b50c8eeaf314) should address the latest request.

**Comment by jgromes** [comment]
> is it possible to just call `RF69::begin` within `SX1X1231::begin`

Unforutnately it's not that easy - SX1231 has different values in the version register. `RF69::begin` would have to be generalized, a bit like  `SX127x::begin`

**Comment by jgromes** [comment]
Looks good, thanks!

**Pull request merged**